packages/node/test/integration/get-app-instance.spec.ts

```diff
@@ -4,7 +4,7 @@ import { Node as NodeTypes } from "@counterfactual/types";
 
 import { Node } from "../../src";
 
-import { setup } from "./setup";
+import { setupWithFirebaseServiceFactory } from "./setup";
```

Comment:
On second thought, why are we using Firebase for local Node testing? Why can't we just use in-memory messaging and storage?

Reply:
Yeah we could. I'll remove this.